pages/pipelines/trigger_step.md

```diff
@@ -239,3 +239,7 @@ steps:
 
 ```
 {: codeblock-file="pipeline.yml"}
+
+> 📘 A note on triggered pipeline dependencies and cancellation
+> When you have multiple pipelines that trigger a build with the same commit on an additional pipeline, while also having <code>Cancel Intermediate Builds</code> enabled, the newest build will be the only build that runs. 
+> So if you have <b>Pipeline A</b> trigger a build with a commit on <b>Pipeline B</b> and <b>Pipeline C</b> trigger a build with the <b>same</b> commit on <b>Pipeline B</b>, whichever pipeline triggered the <b>first</b> build will be <b>cancelled</b> in favour of the newest triggered build.
```

Comment:
Thanks very much for your contribution @HugeIRL !

Just a few points:

- As indicated in our [Markdown syntax style guide](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#callouts), it is better to use Markdown syntax within callouts as they make the source more readable.
- As indicated in the accompanying [Writing style guide](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#ui-elements), note the difference between UI elements and emphasis.
- I removed some additional spacing at the end of a line, which was causing the linter to fail.

```suggestion

> 📘 A note on triggered pipeline dependencies and cancellation
> When you have multiple pipelines that trigger a build with the same commit on an additional pipeline, while also having `Cancel Intermediate Builds` enabled, the newest build will be the only build that runs.
> So if you have **Pipeline A** trigger a build with a commit on **Pipeline B** and **Pipeline C** trigger a build with the _same_ commit on **Pipeline B**, whichever pipeline triggered the _first_ build will be _cancelled_ in favor of the newest triggered build.
```

Reply:
I'm assuming **Cancel Intermediate Builds** (verbatim) refers to something in the Buildkite UI?

Reply:
Yep! That's correct. Your suggestions make sense - on that note though I don't think we should merge this PR until we get the words included for the missing example here too. I struggled with explaining the concept as part of this PR, and require some magic hands here to help us get the words going for the examples. I think maybe having this as a "note" section isn't ideal, and instead it should be it's own sub section of content in some capacity with the two examples living there, as the distinctions here are important to convey in the docs, as they are not easily conveyable in a live discussion with folks without detailed explanation, and some confusion.

Reply:
Thanks @HugeIRL , yes that's a great point!
I'll work your examples above into this PR and will extract the information from the callout and rework this into its next revision.

Reply:
Hey @gilesgas just bumping this so it's not lost.

Reply:
Sorry for my delay on this one @HugeIRL , this PR's definitely on my radar!
I'll try looking at this today and incorporate your example into this PR.

Reply:
Hi @HugeIRL , I've finally got around to addressing this. Sorry it's taken a while.

I decided to explain everything around the first example you provided, and so I hope my interpretation is correct. Please let me know so I can approve this PR...

The revised content can be [previewed here](https://2807--bk-docs-preview.netlify.app/docs/pipelines/trigger-step#a-note-on-triggered-pipeline-dependencies-and-cancellation).

Reply:
Yeah I originally liked the note call out idea, but now that its getting larger, we should probably move it out.

An idea I would have is that the note/callout explains the high level overview of what we're describing, and then maybe under the callout we go into more detail?